roles/openshift_prometheus/templates/prometheus_deployment.j2

```diff
@@ -208,7 +208,7 @@ spec:
         secret:
           secretName: prometheus-tls
       - name: prometheus-data
-{% if openshift_prometheus_storage_type == 'pvc' %}
+{% if openshift_prometheus_storage_type == 'nfs' %}
```

Comment:
I'm a little confused here.  Storage type refers to the backing store and now how it is abstracted in the deployment?  Conceptually, couldn't my claim be of type hostpath where the PVC is bound to something local.  Or could we not define the claim to use something other then NFS?  As implemented this means we ONLY allow use of PVC and NFS combo.

Reply:
@jcantrill I am also a bit confused.
I thought that was what you meant here: https://bugzilla.redhat.com/show_bug.cgi?id=1495446#c9

Another way we can go is keep both variables separately:
```
openshift_prometheus_storage_kind = [nfs|hostpath|dynamic|etc..] 
openshift_prometheus_storage_type = [pvc]
```
default for both is None.
```
if openshift_prometheus_storage_kind == pv_storage_kind*
  openshift_prometheus_storage_type <= 'pvc'
```

pv_storage_kind* => I need a list of possible values.

How does that solution look?